build-tools/build-image/cross/Dockerfile

```diff
@@ -50,13 +52,11 @@ RUN mkdir -p /usr/local/src/protobuf \
   && protoc --version
 
 # Use dynamic cgo linking for architectures other than amd64 for the server platforms
-# More info here: https://wiki.debian.org/CrossToolchains
-RUN echo "deb http://emdebian.org/tools/debian/ jessie main" > /etc/apt/sources.list.d/cgocrosscompiling.list \
-  && curl -s http://emdebian.org/tools/debian/emdebian-toolchain-archive.key | apt-key add - \
-  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do dpkg --add-architecture ${platform}; done \
+# To install crossbuild essential for other architectures add the following repository.
+RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/sources.list.d/cgocrosscompiling.list \
   && apt-get update \
-  && apt-get install -y build-essential \
-  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do apt-get install -y crossbuild-essential-${platform}; done \
+  && apt-get install -y --force-yes  build-essential \
+  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do apt-get install -y --force-yes crossbuild-essential-${platform}; done \
```

Comment:
it looks like if you run `apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 40976EAF437D05B` before `apt-get update` you don't need the `--force-yes`.